README.md

```diff
@@ -34,14 +34,24 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. If you were to describe semantic HTML to the next cohort of students, what would you say?
 
+It's an easy way of describing to the browser what that specific content is because the browser doesn't look at the class and id labels to quickly pull information about the page. 
```

Comment:
Great answer! this describes semantic html very well and i like how you included that styling attributes are not accounted as semantic